src/adapter/base.cr

```diff
@@ -31,6 +31,10 @@ abstract class Granite::Adapter::Base
     yield @database
   end
 
+  def log(query : String, params = [] of String) : Nil
+    Granite::ORM.settings.logger.info "#{query}: #{params}"
```

Comment:
maybe not default to `logger.info` since this will always will output the severity `(info)`?